src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java

```diff
@@ -155,8 +157,8 @@ public void addNotifications(ProgrammeMembership programmeMembership)
           Date when = getScheduleDate(startDate, daysBeforeStart);
 
           JobDataMap jobDataMap = new JobDataMap();
-          jobDataMap.put("tisId", programmeMembership.getTisId());
-          jobDataMap.put("personId", programmeMembership.getPersonId());
+          jobDataMap.put(TIS_ID_FIELD, programmeMembership.getTisId());
+          jobDataMap.put(PERSON_ID_FIELD, programmeMembership.getPersonId());
           jobDataMap.put("programmeName", programmeMembership.getProgrammeName());
```

Comment:
This awkward mix of constants and fixed strings will be resolved in the next slice.